README.md

```diff
@@ -52,11 +43,20 @@ def fib(n: int) -> Iterator[int]:
         yield a
         a, b = b, a + b
 ```
+See [the documentation](http://mypy.readthedocs.io/en/stable/introduction.html) for more examples.
+
+For Python 2.7, the standard annotations are written as comments:
+```python
+def reverse(s):
+    # type: str -> str
+    return s[::-1]
```

Comment:
Maybe use a slightly less trivial example? Like ``is_palindrome(s)`` that returns ``s == s[::-1]``?

Reply:
Nice! :)
I tried to find something like that and all I came up with was reverse :(